### ISSUE TITLE
Migration doesn't seem to contain group table information as yet

### ISSUE THREAD

**Issue by willcrobinson** [santigarcor/laratrust#53]
Following setup instructions to the letter and seeing this error from MySQL:

```SQLSTATE[42S22]: Column not found: 1054 Unknown column 'role_user.group_id' in 'field list' (SQL: select `roles`.*, `role_user`.`user_id` as `pivot_user_id`, `role_user`.`role_id` as `pivot_role_id`, `role_user`.`group_id` as `pivot_group_id` from `roles` inner join `role_user` on `roles`.`id` = `role_user`.`role_id` where `role_user`.`user_id` = 1) ```

Using 3.1.0-rc.1.

**Comment by KKSzymanowski** [comment]
Where does this error occur? Can you paste the stack trace?

**Comment by santigarcor** [comment]
@willcrobinson could you please give a more detailed description of the error?

**Comment by willcrobinson** [comment]
I installed a fresh copy of Laravel 5.3 and followed the setup instructions for LaraTrust 3.1.0-rc.1. On every page that I tried to use LaraTrust's blade directives I got the SQL error shown above. I opened the migration file that the setup file generated and there was no mention of groups anywhere within it (which is why I guess it's throwing up the database error about being unable to find `role_user.group_id`.

I've checked the migration code in the master branch and can't see that groups are referenced anywhere either. Am I missing something? https://github.com/santigarcor/laratrust/blob/3.0/src/commands/MigrationCommand.php

TL;DR: role_user.group_id doesn't exist and none of the setup files or migration files seem to create any group tables or fields.

**Comment by santigarcor** [comment]
It's weird I remember putting that, but I'll check that.

Thanks for the help 😀

**Comment by santigarcor** [comment]
Did you used a previous version of laratrust? because i checked the `migration.blade.php` file and it has all the groups tables and columns you can see it right here https://github.com/santigarcor/laratrust/blob/master/src/views/generators/migration.blade.php

And the laratrust config file has the groups information too: https://github.com/santigarcor/laratrust/blob/master/src/config/config.php
